File: 백준/Bronze/29722. 브실혜성/브실혜성.py
y, m, d = map(int, input().split('-'))
n = int(input())

total = y * 360 + (m-1) * 30 + (d-1) + n

ny, total = divmod(total, 360)
nm, nd = divmod(total, 30)

print(f"{ny:04d}-{nm+1:02d}-{nd+1:02d}")

File: 백준/Bronze/29722. 브실혜성/브실혜성.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] ymd = Arrays.stream(br.readLine().split("-"))
                .mapToInt(Integer::parseInt)
                .toArray();
        int y = ymd[0];
        int m = ymd[1];
        int d = ymd[2];

        int total = y * 360 + (m - 1) * 30 + (d - 1) + Integer.parseInt(br.readLine());

        int ny = total / 360;
        total %= 360;
        int nm = total / 30;
        int nd = total % 30;

        System.out.printf("%04d-%02d-%02d%n", ny, nm+1, nd+1);
    }
}